src/ShoppingList.js

```diff
@@ -28,6 +29,16 @@ class ShoppingList extends Component {
     e.preventDefault();
   }
 
+  deleteItem(key){
+    var filteredItems = this.state.items.filter(function (item) {
```

Comment:
This can be `const` as the intent is to not reassign it:
```suggestion
    const filteredItems = this.state.items.filter(function (item) {
```